lms/templates/ccx/coach_dashboard.html

```diff
@@ -13,6 +13,12 @@
   <%static:css group='style-vendor-tinymce-content'/>
   <%static:css group='style-vendor-tinymce-skin'/>
   <%static:css group='style-course'/>
+
+  <script src="${static.url('xmodule_js/common_static/js/vendor/jquery.event.drag-2.2.js')}"></script>
+  <script type="text/javascript" src="${static.url('js/vendor/slick.core.js')}"></script>
+  <script type="text/javascript" src="${static.url('js/vendor/slick.grid.js')}"></script>
```

Comment:
No. No, no, no. Javascript should be loaded using require.js, which allows us to manage dependencies between scripts. Require.js also allows us to minify and optimize Javascript before it is served on production. Do NOT add Javascript with `<script>` tags.